the-list-be/server.js

```diff
@@ -66,9 +66,10 @@ router.patch('/students/:id', async ctx => {
   ctx.body = editNameStudentResult;
 });
 
-router.delete('/students/:id', async ctx => {
+router.delete('/students/', async ctx => {
```

Comment:
Лучше
```suggestion
router.delete('/students/:id', async ctx => {
```

`body` обычно передают в `POST`, `PUT`, и `PATCH` запросах. Идентификатор ресурса во всех этих запросах чаще помещают в путь. Если нет причин не следовать стандартам, лучше их соблюдать. Так поддерживать продукт в дальнейшем будет проще.